### PR TITLE
Changes for subscription to project or everything

### PR DESCRIPTION
**Pull request by VineetBala-AOT**
*Description of changes:*
- Changes on email verification to send a response on create
- Back end changes on subscription to store subscription option
- Front end changes to add the radio option on subscribe

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

**Review by djnunez-aot — APPROVED** [review]

File: met-api/src/met_api/constants/subscription_type.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+"""Constants for subscription."""
+from enum import IntEnum
+
+
+class SubscriptionType(IntEnum):
+    """Enum of Subscription Type."""
+
+    ENGAGEMENT = 1
+    PROJECT = 2
+    TENANT = 3
```
Comment (saravanpa-aot) [inline met-api/src/met_api/constants/subscription_type.py:23]: please write some comments about what does it do..
like what is subscription type and what each type does..

File: met-api/src/met_api/resources/email_verification.py [hunk 2]
```diff
@@ -79,8 +79,8 @@ def post():
         try:
             requestjson = request.get_json()
             email_verification = EmailVerificationSchema().load(requestjson)
-            EmailVerificationService().create(email_verification)
-            return {}, HTTPStatus.OK
+            email_verification = EmailVerificationService().create(email_verification)
+            return email_verification, HTTPStatus.OK
```
Comment (saravanpa-aot) [inline met-api/src/met_api/resources/email_verification.py:83]: same variable name?

File: met-web/src/models/subscription.ts [hunk 3]
```diff
@@ -1,3 +1,12 @@
+export interface CreateSubscription {
```
Comment (saravanpa-aot) [inline met-web/src/models/subscription.ts:1]: do we name our interrfaces as noun?
is something like SubscriptionCreationData better or SubscriptionData

**Review by saravanpa-aot — APPROVED** [review]

File: met-api/src/met_api/constants/subscription_type.py (hunk 1 above)
Comment (VineetBala-AOT) [reply met-api/src/met_api/constants/subscription_type.py]: Added the comments.

File: met-api/src/met_api/resources/email_verification.py (hunk 2 above)
Comment (VineetBala-AOT) [reply met-api/src/met_api/resources/email_verification.py]: Changed, thanks.

File: met-web/src/models/subscription.ts (hunk 3 above)
Comment (VineetBala-AOT) [reply met-web/src/models/subscription.ts]: Renamed, thanks.

**Pull request merged**